locale/core.yml

```diff
@@ -628,11 +633,6 @@ core:
       submit_button: => core.ref.save_changes
       title: => core.ref.reset_your_password
 
-    # Translations in this namespace are used to format page meta titles.
-    meta_titles:
-      with_page_title: "{pageNumber, plural, =1 {{pageTitle} - {forumName}} other {{pageTitle}: Page # - {forumName}}}"
-      without_page_title: "{pageNumber, plural, =1 {{forumName}} other {Page # - {forumName}}}"
-
```

Comment:
Is this not a breaking change for language packs?

Reply:
Hmm that's a tough one. As per the definition in https://discuss.flarum.org/d/26242-draft-public-api-policy, our public API is defined in terms of the interfaces of the code, not necessarily of behavior. This will require language packs to update, but in the meantime it won't break forums using those language packs, so I don't see it as illegal.

Reply:
It's fine, we introduced these in this cycle anyway, so it doesn't matter, it never made it to a release.

Reply:
The argument is that even requiring translations means that extensions will need to add translations to avoid showing the 'core.lib' text. However, since that's easily addressable and doesn't break functionality, I think it's fine.

Reply:
I do agree with that.

Reply:
Sounds good, I figured it wasn't that big of a deal as it's not breaking.